### PR TITLE
feat: add limit option to influx bucket list

### PR DESCRIPTION
**Pull request by russorat**
Closes https://github.com/influxdata/influxdb/issues/19490

Describe your proposed changes here.

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Feature flagged (if modified API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: cmd/influx/bucket.go [hunk 1]
```diff
@@ -179,6 +183,13 @@ func (b *cmdBucketBuilder) cmdList() *cobra.Command {
 			EnvVar: "BUCKET_NAME",
 			Desc:   "The bucket name",
 		},
+		{
+			DestP:  &b.limit,
+			Flag:   "limit",
+			Short:  'l',
+			EnvVar: "LIMIT",
+			Desc:   "The max number of results to return (must be < 100)",
```
Comment (danxmoran) [inline cmd/influx/bucket.go:191]: We should probably add this to all the find / list commands

Comment (danxmoran) [reply cmd/influx/bucket.go:191]: What happens if a user passes > 100? Does the server just return the first 100 things, or does something go boom?

Comment (russorat) [reply cmd/influx/bucket.go:191]: they get a message: `Error: Failed to retrieve buckets: limit must be between 1 and 100.`

**Comment by russorat** [comment]
I agree we should add this to all list commands. I'm not skilled enough to do that very well though.

**Comment by russorat** [comment]
i'm going to add this in a few more places, and add an offset option as well

**Comment by wbaker85** [comment]
Closing this since https://github.com/influxdata/influx-cli/pull/307 implemented bucket pagination in the CLI.

**Pull request closed without merging**